mitmproxy/proxy/layers/dtls.py

```diff
@@ -0,0 +1,175 @@
+import struct
```

Comment:
I was considering "merging" the DTLS layers with the TLS layers. We could add a "mode" or flag that tells the TLS layer that it is in DTLS mode. This would required that we use if-statements to call DTLS related code which is almost insignificant compared to all the logic the layers share. Thoughts?

Reply:
We could have something like in this file:
```python
class ClientDTLSLayer(ClientTLSLayer):
       def __init__(self, context: context.Context):
         super().__init__(context, dtls_mode=True)
```